src/templates/finops-hub/main.bicep

```diff
@@ -29,6 +29,13 @@ param exportRetentionInDays int = 0
 @description('Optional. Number of months of cost data to retain in the ingestion container. Default: 13.')
 param ingestionRetentionInMonths int = 13
 
+@description('Optional. Remote storage account for ingestion dataset.')
+param hubStorageAccountUri string = ''
```

Comment:
nit: Can we be more explicit about this being the remote storage account? Maybe `remoteHubStorageUri`? Same for the key.